modules/ROOT/pages/npm-projects.adoc

```diff
@@ -33,9 +33,71 @@ The location of the main {productname} script will be: `+node_modules/tinymce/ti
 * Using a webserver route, or
 * Copying the `+tinymce+` directory to a public folder using a build tool such as Gulp or Webpack.
 
+[[install-premium-plugins]]
+== Install premium plugins
+
+The `+tinymce-premium+` NPM package provides all premium plugins, skins, themes, icons, and language packs for commercial TinyMCE customers.
```

Comment:
```suggestion
The `+tinymce-premium+` NPM package provides all premium plugins, skins, icons, and language packs for commercial TinyMCE customers.
```

Reply:
Resolved in [this commit](https://github.com/tinymce/tinymce-docs/pull/3928/commits/85a88c5d10dc4c8e3a2b2780fc45778b20ed0be7)